### ISSUE TITLE
Wrong Tag in statement.py

### ISSUE THREAD

**Issue by davidrolli** [VerenigingCampusKabel/python-sepa#5]
After many hours of search I found the wrong written tag in the file 'statement.py', This had important consequences in my further data processing because of the lack of important text in the field 'AdditionalTransactionInformation'.

You have to change in the file 'statement.py' in line 208 and 409 the tag 'AddtTxInf' (wrong) to 'AddtlTxInf' (correct).

Greetings,  David Rolli

**Comment by DanielleHuisman** [comment]
Changed in https://github.com/VerenigingCampusKabel/python-sepa/commit/e1b4c1e98d5e6cef9375573453d28ff5e00e1eda. I will make a new release tonight.

**Comment by AMTQ** [comment]
I tried correcting this by hand, it seems to ignore it and still does not output the AddtlTxInf anymore. Before it did, as I remember. Any other ideas where this might come from? Also I saw that my bank now uses version camt.053.001.**08**, however in the spec it does not mention any changes to AddtlTxInf. https://www.six-group.com/dam/download/banking-services/interbank-clearing/de/standardization/iso/swiss-recommendations/implementation-guidelines-camt-developer_support-2022.pdf

**Comment by DanielleHuisman** [comment]
Sorry for the delay, but I just published version 0.5.4 to PyPI. The `AddtlTxInf` tag works just fine for me with a [sample file](https://developer.gs.com/docs/services/transaction-banking/camt53-sample/):

```python
import json
import re
from sepa import parser

file_path = './camt-053-example.xml'


def strip_namespace(xml):
    return re.sub(' xmlns="[^"]+"', '', xml, count=1)


# Read file
with open(file_path, 'r') as f:
    input_data = f.read()

# Parse the bank statement XML to JSON
camt53 = parser.parse_string(parser.bank_to_customer_statement, bytes(strip_namespace(input_data), 'utf8'))

# Pretty print JSON information from bank statement
print(json.dumps(camt53, indent=4))
print()
```

```json
...
                            "transaction_details": [
                                {
                                    "refs": {
                                        "message_id": "GSFCY3XIA1XR",
                                        "account_servicer_reference": "GPB2121501121577",
                                        "payment_information_id": "WB/GS/Msg002/WB01002",
                                        "transaction_id": "GPB2121501121577"
                                    },
                                    "amount_details": {
                                        "instructed_amount": {
                                            "amount": {
                                                "_value": "36.80",
                                                "currency": "USD"
                                            }
                                        },
                                        "transaction_amount": {
                                            "amount": {
                                                "_value": "36.80",
                                                "currency": "USD"
                                            }
                                        }
                                    },
                                    "related_parties": {},
                                    "purpose": {
                                        "proprietary": "HSA-FUND"
                                    },
                                    "additional_information": "SEC Code: CCD"
                                }
                            ]
...

**Comment by BenjoNX** [comment]
I developed a soft based that could generate a xml sepa file [sepaxml.com](https://www.sepaxml.com) available in multi-linguage
